### PR TITLE
Fix for huge payload fetched from API on dashboard/configuration/plugins view

### PR DESCRIPTION
**Pull request by kamilpastuszka**
I want to merge this in order to get rid of unnecessary payload that would make page load faster

**PR intended to be tested with API branch:** 

### Pull Request Checklist

1. [ ] This code contains UI changes
2. [ ] All visible strings are translated with proper context including data-formatting
3. [ ] Attributes `[data-test-id]` are added for new elements
4. [ ] Changes are mentioned in the changelog
5. [ ] The changes are tested in different browsers and in light/dark mode

### Test environment config

API_URI=https://qa.staging.saleor.cloud/graphql/

### CODE REVIEW

**Comment by patrys** [comment]
Task linked: [SALEOR-3657 Huge payload fetched from API on dashboard/configuration/plugins view](https://app.clickup.com/t/2549495/SALEOR-3657)

File: src/fragments/plugins.ts [hunk 1]
```diff
@@ -10,9 +10,20 @@ export const configurationItemFragment = gql`
...
+      name
+      slug
+    }
+  }
+`;
+
+export const pluginConfigurationExtendedFragment = gql`
   ${configurationItemFragment}
-  fragment PluginConfigurationFragment on PluginConfiguration {
+  fragment PluginConfigurationExtendedFragment on PluginConfiguration {
```
Comment (jwm0) [inline src/fragments/plugins.ts:26]: This extended fragment could use `pluginConfigurationBaseFragment` as a base:
```
${configurationItemFragment}
${pluginConfigurationBaseFragment}
fragment PluginConfigurationExtendedFragment on PluginConfiguration {
  ...PluginConfigurationBaseFragment
  configuration {
      ...ConfigurationItemFragment
  }
}
```

Comment (kamilpastuszka) [reply src/fragments/plugins.ts:27]: That's been changed now

**Review by jwm0 — APPROVED** [review]

**Review by dominik-zeglen — CHANGES_REQUESTED** [review]
You forgot to run `npm run build-types`

**Review by dominik-zeglen — APPROVED** [review]

**Pull request merged**